types_generate.go

```diff
@@ -272,6 +277,12 @@ func main() {
 					fields = append(fields, f)
 					continue
 				}
+				if t == "SVCBKeyValue" {
```

Comment:
This should probably check for ``` `dns:"pairs"` ``` rather than `"SVCBKeyValue"`.

Reply:
It needs to be consistent with the previous 8 lines of code?

Reply:
I'd rather this check for `dns:"pairs"`, the others probably should be changed at some later date to use tags as well instead. It might be better to leave this as is for now and do something in a later PR. I'll leave it up to you.